server/odc-service/src/main/java/com/oceanbase/odc/metadb/dbobject/DBColumnRepository.java

```diff
@@ -66,7 +68,12 @@ default List<DBColumnEntity> batchCreate(List<DBColumnEntity> entities) {
                 .add(DBColumnEntity::getObjectId)
                 .add(DBColumnEntity::getOrganizationId)
                 .build();
-        return batchCreate(entities, sql, getter, DBColumnEntity::setId, 200);
+        Iterable<List<DBColumnEntity>> partitions = Iterables.partition(entities, batchSize);
```

Comment:
`DBSchemaAccessorUtil#partitionFind` will help you do this

Reply:
Good reminder, I refined it.